File: 2015/01/01.go
package main

import (
	"fmt"
	"os"
	"strings"
)

func main() {

	raw_data, _ := os.ReadFile("./input.txt")
	problem_input := string(raw_data)

	open_parentheses := strings.Count(problem_input, "(")
	closed_parentheses := strings.Count(problem_input, ")")
	difference := open_parentheses - closed_parentheses
	
	fmt.Println(difference)

}

File: 2015/01/01.js
const fs = require('fs')

const problem_input = fs.readFileSync('./input.txt', 'utf8');
const open_parentheses = problem_input.match(/\(/g).length;
const closed_parentheses = problem_input.match(/\)/g).length;
const difference = open_parentheses - closed_parentheses;

console.log(difference);
